src/common/components/vanilla-modules/forms/form-group-input.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+
+import styles from '../../../style/vanilla/css/forms.css';
+
+export default function FormGroupInput(props) {
+  const { disabled, name, label, type, placeholder } = props;
+  const id = `ID_${name}`;
+  const status = props.touched ? ( props.valid ? 'p-form-validation is-success' : 'p-form-validation is-error') : null;
+
+  return (
+    <div className={ `${styles['p-form__group']} ${styles[status]}` }>
+      <label htmlFor={ id } className={ styles['p-form__label'] }>
+        { label }
+      </label>
+      <div className={ styles['p-form__control'] }>
+        <input
+          id={ id }
+          name={ props.sensitive ? null : name }
+          data-name={ name }
+          type={ type }
+          required={ props.required }
+          disabled={ disabled }
+          placeholder={ placeholder }
+          onChange={ props.onChange }
+          onBlur={ props.onBlur }
+          value={ props.value || '' }
+        />
+        { props.errorMsg &&
+          <p className={ styles['p-form-validation__message'] } role="alert">
+            <strong>Error:</strong> { props.errorMsg }
+          </p>
+        }
+      </div>
+    </div>
+  );
+}
+
+FormGroupInput.propTypes = {
+  name: PropTypes.string,
+  label: PropTypes.string,
+  placeholder: PropTypes.string,
+  required: PropTypes.bool,
+  disabled: PropTypes.bool,
```

Comment:
I guess we did it like that before as well, but right now as a convention I'd rather name bool props as `isDisabled`.

On the other hand with props like this (that link directly to a DOM property) it's quite clear to have it as `disabled`, as it speaks for itself.
So I'm not exactly sure what to do with that, but kinda opening a discussion ;)